src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicClientTransformer.java

```diff
@@ -218,6 +222,26 @@ private StaticLangXSettingsView generateXSettings(SurfaceTransformerContext cont
     return settingsMembers;
   }
 
+  private List<ReroutedGrpcView> generateReroutedGrpcView(SurfaceTransformerContext context) {
+    SurfaceNamer namer = context.getNamer();
+    ModelTypeTable typeTable = context.getTypeTable();
+    Set<ReroutedGrpcView> rerouteds = new HashSet<>();
```

Comment:
And perhaps use a LinkedHashSet so the iteration order is predictable? We wouldn't want generating the code twice to give different results. (I'm guessing the order is used later - I haven't checked yet.)